samples/src/main/java/com/backblaze/b2/sample/B2Sample.java

```diff
@@ -251,12 +251,7 @@ private static void mainGuts(PrintWriter writer,
         B2FileVersion file2ByName = client.getFileInfoByName(bucket.getBucketName(), file2.getFileName());
         writer.println("file2ByName: " + file2ByName);
 
-        if (!file2again.getFileId().equals(file2ByName.getFileId()) ||
-            file2again.getContentLength() != file2ByName.getContentLength() ||
-            !file2again.getContentType().equals(file2ByName.getContentType()) ||
-            !file2again.getContentSha1().equals(file2ByName.getContentSha1()) ||
-            file2again.getUploadTimestamp() != file2ByName.getUploadTimestamp() ||
-            !file2again.getFileName().equals(file2ByName.getFileName())) {
+        if (!file2again.equals(file2ByName)) {
```

Comment:
thanks.
i merged the fix into master a few minutes ago.
if you pull that, does this pass now?

thanks,
ab